From 38fc95dd6797e785e0db715cf8b8ee24a9cee9cd Mon Sep 17 00:00:00 2001
From: Valentin David <me@valentindavid.com>
Date: Sun, 26 Oct 2025 11:07:13 +0100
Subject: [PATCH 7/8] openssh: Add missing dependency to libfido2

This is needed to use fido2 cards as ssh key.
---
 elements/components/openssh.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/components/openssh.bst b/elements/components/openssh.bst
index 240615a00..87e8076a7 100644
--- a/elements/components/openssh.bst
+++ b/elements/components/openssh.bst
@@ -2,6 +2,7 @@ kind: autotools
 
 build-depends:
 - public-stacks/buildsystem-autotools.bst
+- components/libfido2.bst
 - components/linux-pam.bst
 - components/openssl.bst
 
-- 
2.51.0

